import-service/handlers/importFileParser.ts

```diff
@@ -0,0 +1,62 @@
+import { APIGatewayProxyHandler, S3Event } from 'aws-lambda';
+import * as AWS from 'aws-sdk'
+import 'source-map-support/register';
+import * as csvParser from 'csv-parser';
+import { BUCKET_PARAMS } from '../utils/constants';
+import { makeErrorResponse, makeCreatedResponse } from '../utils/response';
+
+export const importFileParser: APIGatewayProxyHandler = async (event: S3Event, _context) => {
```

Comment:
`S3Handler` would be better. The handler should not return an HTTP response.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b9b6b0e9bd5de6ed18be0a3e07816f60fe6f2cfa/types/aws-lambda/trigger/s3.d.ts#L3